src/js/auth.js

```diff
@@ -0,0 +1,41 @@
+const { nanoid } = 'nanoid';
+
+const auth = () => {
+  const getSession = () => {
+    const sessionData = sessionStorage.getItem('session');
```

Comment:
Let's use a more complex key here, session might be too basic.

What of?
```suggestion
    const sessionData = sessionStorage.getItem('moshoodabdullahi-todo-list-app-session');
```

Reply:
Done